lib/ember_cli/shell.rb

```diff
@@ -37,6 +37,7 @@ def install
         exec "#{paths.bundler} install"
       end
 
+      exec "#{paths.ember} version || rm -rf #{paths.npm_deps} #{paths.bower_deps}"
```

Comment:
The exception would be `SystemExit`... seems kinda hacky. What do you think about adding a `Runner#run` method, which does the same thing as `#run!`, but returning false instead of raising `SystemExit`?